resources/css/aphrodite-shortcomings.css

```diff
@@ -6,12 +6,79 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
 *, :before, :after {box-sizing: border-box;}
 strong {font-weight: bold;}
 
 /* See https://github.com/Khan/aphrodite/issues/30 */
+
 .episode-page {
   &__episode-transcript p {
     margin-bottom: 20px
   }
 }
+
+.container {
```

Comment:
Do these styles have to be nested in `.container`? I'd love to avoid nesting as much as possible if we can.